shell/app/modules/project/pages/test-manage/case/manual-test.tsx

```diff
@@ -46,6 +46,7 @@ const ManualTest = () => {
   const [enhanceFilterVisible, setEnhanceFilterVisible] = React.useState(false);
   const [showRefresh, setShowRefresh] = React.useState(false);
   const [justImportSetId, setJustImportSetId] = React.useState<number | null>(null); // to cache the test set id which current user just Import
+  const [importExportRecordKey, setImportExportRecordKey] = React.useState<number>(0);
```

Comment:
if u pass `0` as init value ,  ts will infer this state as number, no need to add generic